tests/integration/test_vulnerability_detector/test_providers/test_providers_os.py

```diff
@@ -43,35 +47,30 @@
 ids = [f"{item['PROVIDER']}-{item['OS']}" for item in params]
 
 # configuration data
-configurations = load_wazuh_configurations(
-    configurations_path,
-    __name__,
-    params=params,
-    metadata=metadata
-)
+configurations = load_wazuh_configurations(configurations_path, __name__, params=params, metadata=metadata)
 
 
 # fixtures
-
 @pytest.fixture(scope="module", params=configurations, ids=ids)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
 
+# tests
```

Comment:
This one neither.

Reply:
Done in 7fcdc7ebecdd546a4957814ec960f6304685d265